.github/workflows/build_devtool.yml

```diff
@@ -0,0 +1,49 @@
+name: Build ROHD devtool
+on:
+  push:
+    branches:
+      - main
+
+# Top-level default, no permissions
+permissions: {}
+
+jobs:
+  build-devtool:
```

Comment:
I think it's still important to have the build-devtool as part of the CI for every branch, we just want to limit the *push to the other branch* to only when there's an actualy merge into main.

One way I think you can do this is leaving it in the general.yml, but adding some conditions on when it should run vs. be skipped: https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution . The push can be a separate step which depends on the previous step.

An easier solution might just be to keep this file as-is, but add back the build alone into the general.yml CI flow.  There's a little duplication, but it's simpler to set up and see what's happening I think.

Reply:
Yup. I think I will just go with the easier option for now. We can add this as a good first issue.